### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by erogluorhan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by erogluorhan** [comment]
@conda-forge-admin, please rerender

**Comment by erogluorhan** [comment]
Hi @khallock could you please help me update the maintainers list for this repo?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -103,8 +103,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - khallock
-    - marylhaley
-    - david-ian-brown
-    - ocefpaf
-    - xylar
+    - erogluorhan
```
Comment (ocefpaf) [inline recipe/meta.yaml:106]: Please do not remove folks without their consent. I certainly do not want to be removed as the original author of this recipe.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ source:
     - grib2.patch
 
 build:
-  number: 23
+  number: 24
```
Comment (ocefpaf) [inline recipe/meta.yaml:14]: No need to bump the recipe build number for this.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -103,8 +103,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - khallock
-    - marylhaley
-    - david-ian-brown
-    - ocefpaf
-    - xylar
```
Comment (khallock) [inline recipe/meta.yaml:110]: I would keep @ocefpaf and @xylar on here for sure, I believe they are both still active in maintaining this package.

Comment (ocefpaf) [reply recipe/meta.yaml:110]: Barely maintaining, sure, but we are doing what we can to keep the builds alive :smile:

Comment (erogluorhan) [reply recipe/meta.yaml:110]: Sure, sorry @ocefpaf for missing that! I just didn't want to bother people who are not officially in the maintaining position. How about keeping you and the others, @khallock ?

Comment (ocefpaf) [reply recipe/meta.yaml:110]: Usually we ask folks to either remove themselves or ask to be removed. The reason is b/c once you do remove them, they will lose access to the feedstock and can no longer merge PR. If they are inactive is not a big deal, specially if they silenced the notifications.

File: recipe/meta.yaml (hunk 1 above)
Comment (erogluorhan) [reply recipe/meta.yaml:111]: Restored the maintainers.

File: recipe/meta.yaml (hunk 2 above)
Comment (erogluorhan) [reply recipe/meta.yaml]: Reverted the bumping.

File: recipe/meta.yaml (hunk 3 above)
Comment (erogluorhan) [reply recipe/meta.yaml]: Restored the original maintainers list

**Comment by ocefpaf** [comment]
@erogluorhan I'm assuming you are substituting @khallock, right? Do you want to try to fix the recipe or do you want me to merge this so you can iterate later in another PR?

**Comment by erogluorhan** [comment]
> @erogluorhan I'm assuming you are substituting @khallock, right? Do you want to try to fix the recipe or do you want me to merge this so you can iterate later in another PR?

I also kept @khallock in the maintainers, so I am not substituting. If @khallock wants to be removed, then we can do that.  

Let me first spend some time working to fix this recipe before you merge it. I will let you know.

**Comment by xylar** [comment]
@erogluorhan, are you still wanting to be added as a maintainer?  If so, we can make a fresh PR for that.

**Comment by erogluorhan** [comment]
> @erogluorhan, are you still wanting to be added as a maintainer? If so, we can make a fresh PR for that.

Thanks for asking! Yes, please do so (even though I couldn't manage some of my time to be put here recently)

**Comment by xylar** [comment]
closing this via #116

**Comment by zklaus** [comment]
Cheers @erogluorhan, great to have you on board! Do you think you might have time to look at NCAR/ncl#173 for a future release of NCL?

**Comment by erogluorhan** [comment]
> Cheers @erogluorhan, great to have you on board! Do you think you might have time to look at [NCAR/ncl#173](https://github.com/NCAR/ncl/pull/173) for a future release of NCL?

Yes, for a future release, but I don't think it could be sooner (say in these few months).

**Comment by zklaus** [comment]
@erogluorhan, I understand that it will be some time until a new release. I would encourage you to have a look at the patch in the near future. It is really rather small and longstanding PRs tend to get more difficult to integrate. In any case, let's take any further discussion over to NCAR/ncl#173.

**Pull request closed without merging**